wazo_agid/handlers/tests/test_userfeatures.py

```diff
@@ -53,6 +53,7 @@ def setUp(self):
             'XIVO_DST_EXTEN_ID': '983274',
             'XIVO_BASE_CONTEXT': 'default',
             'WAZO_CALL_RECORD_ACTIVE': '0',
+            'WAZO_USER_MOH_UUID': '00000000-feed-dada-iced-c0ffee000000'
```

Comment:
`iced` is not a valid hexadecimal literal :trollface:

Reply:
fixed